app/components/svg-transparent-rect.jsx

```diff
@@ -0,0 +1,75 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+
+const FILTERS = {
+  invert: "url('#svg-invert-filter')"
+};
+
+const INVERT =
```

Comment:
If this is a transparent SVG does it need an invert option?

Reply:
Nope, well spotted